cpp/src/aztec3/circuits/rollup/merge/native_merge_rollup_circuit.cpp

```diff
@@ -55,10 +56,12 @@ void assert_both_input_proofs_of_same_rollup_type(BaseOrMergeRollupPublicInputs
  * @param right - The public inputs of the right rollup (base or merge)
  * @return NT::fr - The height of the rollup subtrees
  */
-NT::fr assert_both_input_proofs_of_same_height_and_return(BaseOrMergeRollupPublicInputs left,
+NT::fr assert_both_input_proofs_of_same_height_and_return(DummyComposer& composer,
+                                                          BaseOrMergeRollupPublicInputs left,
```

Comment:
```suggestion
                                                          BaseOrMergeRollupPublicInputs const& left,
```

Reply:
parameters which won't get modified by a function should often be `const&` (a constant reference)

Reply:
good catch!